can-stache.js

```diff
@@ -476,7 +476,24 @@ function stache (filename, template) {
 				comment: text
 			});
 		},
-		done: function(lineNo){}
+		done: function(lineNo){
+			//!steal-remove-start
+			// warn if closing magic tag is missed #675
+			var last = state.sectionElementStack[state.sectionElementStack.length - 1];
+			if (process.env.NODE_ENV !== 'production') {
+				if(section instanceof HTMLSectionBuilder) {
```

Comment:
I don't think you need this line here.  The block you copied this from (probabaly `makeRendererAndUpdateSection`) was in a function that was passed `section` as a parameter.  In this function you're just referencing the top level `section`, which is always an `HTMLSectionBuilder`.

Reply:
Yeah, I added this check in the 2nd commit the first one was good.

Reply:
is fixed now